Description: Also install 22x22, 36x36 & 42x42 application icons in hicolor theme.
 The 22x22 icon is required for Debian Policy 9.6:
 "
  Unless hidden by default, the desktop entry must point to a PNG or SVG
  icon with a transparent background, providing at least the 22×22 size,
  and preferably up to 64×64. The icon should be neutral enough to
  integrate well with the default icon themes. It is encouraged to ship
  the icon in the default hicolor icon theme directories, or to use an
  existing icon from the hicolor theme.
 "
 https://www.debian.org/doc/debian-policy/ch-opersys.html#s-menus
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://groups.google.com/d/msg/spatialite-users/NqdOCvTatqk/lH3QqpNyNwAJ
Applied-Upstream: https://www.gaia-gis.it/fossil/spatialite_gui/ci/573831584a?sbs=1

--- a/gnome_resource/Makefile.am
+++ b/gnome_resource/Makefile.am
@@ -4,7 +4,7 @@ EXTRA_DIST = spatialite-gui.png spatiali
 desktopdir = $(datadir)/applications
 desktop_DATA = spatialite-gui.desktop
 
-ICON_SIZES = 8 16 24 32 40 48 64 72 80 96 128 192 256 512
+ICON_SIZES = 8 16 22 24 32 36 40 42 48 64 72 80 96 128 192 256 512
 
 install-data-local:
 	@for size in $(ICON_SIZES); do \
